services/storage/signed-uploads/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const slsw = require('serverless-webpack');
+const nodeExternals = require('webpack-node-externals');
```

Comment:
We don't use `webpack-node-externals` in the other services, we only set `externals: [{ 'aws-sdk': true }],`. Last time I checked,  with`webpack-node-externals` enabled, `aws-sdk` was getting packaged with the lambda, which is something we want to avoid. Not sure if it still does it

Reply:
I have taken too old example. Removing it.